timeseries/src/autogluon/timeseries/models/autogluon_tabular/mlforecast.py

```diff
@@ -167,19 +165,10 @@ def _get_mlforecast_init_args(self, train_data: TimeSeriesDataFrame, model_param
             target_transforms.append(Differences(differences))
             self._sum_of_differences = sum(differences)
 
-        scaler_name = model_params.get("scaler")
-        if scaler_name is None:
-            pass
-        elif scaler_name == "standard":
-            self._scaler = StandardScaler()
-        elif scaler_name == "mean_abs":
-            self._scaler = MeanAbsScaler()
-        else:
-            logger.warning(
-                f"Unrecognized `scaler` {scaler_name} (supported options: ['standard', 'mean_abs', None]). Scaling disabled."
-            )
-
-        if self._scaler is not None:
+        # Support "scaler" for backward compatibility
+        scaler_type = model_params.get("target_scaler", model_params.get("scaler"))
+        if scaler_type is not None:
+            self._scaler = MLForecastScaler(scaler_type=scaler_type)
```

Comment:
To discuss: Previously, `DirectTabular` and `RecursiveTabular` had a `"scaler"` hyperparameter that behaved very similarly to `target_scaler`. With the current change, the models "hijack" the `target_scaler` argument and implement custom scaling option (different from what's done in `AbstractTimeSeriesModel`). This is necessary to ensure that scaling is performed after differencing (inside the MLForecast object), which usually results in better accuracy.

Does this make sense?

Reply:
I am not quite sure what "This is necessary to ensure that scaling is performed after differencing" means, but overall it seems plausible to me that we transform the previously-used scaler into an `MLForecastScaler` now.